examples/experimental/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,159 @@
+ limitations under the License.
+"""
+
+import os
+import subprocess
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import openvino.runtime as ov
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow.python.keras.metrics import Metric
+
+import nncf
+from examples.experimental.tensorflow.mobilenet_v2.utils import center_crop
+
+# Path to the `mobilenet_v2` directory.
+ROOT = Path(__file__).parent.resolve()
+# Path to the directory where the original and quantized models will be saved.
+MODEL_DIR = ROOT / 'mobilenet_v2_quantization'
+# Path to ImageNet validation dataset.
+DATASET_DIR = Path("/ssd/tensorflow/imagenet") #ROOT / 'imagenet'
+
+
+def run_example():
+    """
+    Runs the MobileNetV2 quantization example.
+    """
+    # Step 1: Instantiates the MobileNetV2 from the Keras Applications.
+    model = tf.keras.applications.MobileNetV2()
+
+    # Step 2: Create calibration dataset.
+    data_source = create_data_source(batch_size=128)
+
+    # Step 3: Apply quantization algorithm.
+
+    # Define the transformation method. This method should take a data item returned
+    # per iteration through the `data_source` object and transform it into the model's
+    # expected input that can be used for the model inference.
+    def transform_fn(data_item):
+        images, _ = data_item
+        return images
+
+    # Wrap framework-specific data source into the `nncf.Dataset` object.
+    calibration_dataset = nncf.Dataset(data_source, transform_fn)
+
+    # Quantization of the TensorFlow Keras model that was created using
+    # Sequential or Keras Functional API. The `quantize` method expects
+    # a TensorFlow Keras model as input and returns a TensorFlow Keras model
+    # that has been quantized using the calibration dataset.
+    quantized_model = nncf.quantize(model, calibration_dataset)
+
+    # Step 4: Save the quantized TensorFlow Keras model.
+    if not MODEL_DIR.exists():
+        os.makedirs(MODEL_DIR)
+
+    model_name = 'mobilenet_v2'
+    saved_model_dir = MODEL_DIR / model_name
+    quantized_model.save(saved_model_dir)
+    print(f"The quantized model is exported to {saved_model_dir}")
+
+    # Step 5: Run OpenVINO Model Optimizer to convert TensorFlow model to OpenVINO IR.
+    mo_command = f'mo --saved_model_dir {saved_model_dir} ' \
+                 f'--model_name {model_name} --output_dir {MODEL_DIR}'
+    subprocess.call(mo_command, shell=True)
+
+    # Step 6: Compare the accuracy of the original and quantized models.
+    print('Checking the accuracy of the original model:')
+
+    metrics = [
+        tf.keras.metrics.CategoricalAccuracy(name='acc@1'),
+        tf.keras.metrics.TopKCategoricalAccuracy(k=5, name='acc@5')
+    ]
+
+    model.compile(metrics=metrics)
+    results = model.evaluate(data_source)
+    print(f'The original model accuracy@top1: {results[1]:.4f}')
+
+    print('Checking the accuracy of the quantized model:')
+    ie = ov.Core()
```

Comment:
Why do we use IR for the accuracy checking instead of the original framework? I mean, would we change it in the future?